### PR TITLE
ForgeRock Android SDK 4.0.0 release preparation

### PR DESCRIPTION
**Pull request by spetrov**
# JIRA Ticket

[SDKS-2516](https://bugster.forgerock.org/jira/browse/SDKS-2516) ForgeRock Android SDK 4.0.0 Release

### CODE REVIEW

**Review by witrisna — APPROVED** [review]
Looks good to me.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
-- Removed `MANAGE_ACCOUNTS` permission from `forgerock-auth` [SDKS-1367]
-- Removed native SSO support (deprecated `sharedUserId` attribute) [SDKS-2260]
 - Fix for WebAuthn authentication for devices which use full screen biometric prompt [SDKS-2340]
+- Fixed functionality for NetworkCollector [SDKS-2445]
+
+#### Changed
+- `public void WebAuthnRegistrationCallback.register(Node node,FRListener<Void> listener)` to `suspend fun register(context: Context, node: Node)`
+- `public void WebAuthAuthenticationCallback.authenticate(@NonNull Fragment fragment, @NonNull Node node, @Nullable WebAuthnKeySelector selector, FRListener<Void> listener)` to `suspend fun authenticate(context: Context, node: Node, selector: WebAuthnKeySelector = WebAuthnKeySelector.DEFAULT)`
+- `FRAClient.updateAccount` now throws `AccountLockException` upon attempt to update a locked account [SDKS-2166]
+- `OathMechanism.getOathTokenCode()`, `HOTPMechanism.getOathTokenCode()` and `TOTPMechanism.getOathTokenCode()`  now throw `AccountLockException` upon attempt to get an OATH token for a locked account [SDKS-2166]
```
Comment (rodrigoareis) [inline CHANGELOG.md:19]: now throws `AccountLockException`

**Review by rodrigoareis — APPROVED** [review]
Changes looks good to me

File: CHANGELOG.md (hunk 1 above)
Comment (spetrov) [reply CHANGELOG.md]: Fixed! Thanks!

**Pull request merged**